spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+# Spring Data integration Sample
```

Comment:
@josh-wong 
It would be great if you could take a look at the README from an English perspective.